DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -99,12 +100,54 @@ extern "C" {
 /* if present, set the number or read/write threads for file transfers */
 #define IRODS_PARALLEL_FILE_SIZE_THRESHOLD_BYTES "irodsParallelFileSizeThresholdBytes"
 
-
 const static unsigned int DEFAULT_NUMBER_OF_IRODS_READ_WRITE_THREADS = 3;
 const static unsigned int MAXIMUM_NUMBER_OF_IRODS_READ_WRITE_THREADS = 10;
 
 const static std::string CHECKSUM_AVU_NAMESPACE{"GLOBUS"};
 
+// map to translate globus_gfs_command_type_t values into strings for error messages
+static std::map<int, std::string> command_map = {
```

Comment:
Can be marked `const` if it never changes.

Reply:
done